src/Bicep.LangServer/Handlers/BicepDisableLinterRuleHandler.cs

```diff
@@ -0,0 +1,120 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+using System;
+using System.IO;
+using System.Reflection;
+using System.Text;
+using System.Threading;
+using System.Threading.Tasks;
+using Bicep.Core;
+using MediatR;
+using Newtonsoft.Json;
+using Newtonsoft.Json.Linq;
+using OmniSharp.Extensions.JsonRpc;
+using OmniSharp.Extensions.LanguageServer.Protocol;
+using OmniSharp.Extensions.LanguageServer.Protocol.Workspace;
+
+namespace Bicep.LanguageServer.Handlers
+{
+    public class BicepDisableLinterRuleHandler : ExecuteTypedCommandHandlerBase<DocumentUri, string, string>
+    {
+        private const string bicepConfigResourceName = "Bicep.LanguageServer.bicepconfig.json";
+
+        private readonly string DefaultBicepConfig;
+
+        public BicepDisableLinterRuleHandler(ISerializer serializer)
+            : base(LanguageConstants.DisableLinterRuleCommandName, serializer)
+        {
+            var assembly = Assembly.GetExecutingAssembly();
+            var stream = assembly.GetManifestResourceStream(bicepConfigResourceName);
```

Comment:
>GetManifestResourceStream [](http://example.com/codeflow?start=34&length=25)

Since we already have this file as a resource in Bicep.Core (see other comment), we should expose that content through some class in that assembly and just call it here.

Reply:
Makes sense. Updated here:
https://github.com/Azure/bicep/commit/fe5c7a77d213a2beada95c437f88061c592ede4b

Thanks!